explainer.md

```diff
@@ -39,18 +38,12 @@ Possible areas for expansion:
 * Providing badging for sites in a normal web browsing context. The current
   proposal is just for installed apps (designed to show up in the operating
   system shelf area). We could also explore icon badging on the drive-by web.
-  This naturally leads into...
```

Comment:
Don't remove this. We still want to explore this in the future.

In fact, could you add the deleted sentence from below up to here, since we still are considering allowing a character or emoji here, just not in the first version. ("Apps that want to render a small status indicator...")

Also add "calling from a service worker".

Reply:
I've updated this to read that per-tab badging is an area of expansion